components/table/index.js

```diff
@@ -80,7 +80,7 @@ class Table extends Component {
                   eventIndex,
                   events,
                   monthIndex,
-                  weekend,
+                  // weekend,
```

Comment:
you commented out weekend, but didn't put it back or implement it in any other way - now the list is not displaying the weekend days in a compact way. Please fix.

Reply:
upsi, fixed:)

Reply:
Nice!